app/views/suggestions/card/_show.html.erb

```diff
@@ -6,14 +6,20 @@
         <small style='margin-left:10px'>
           <%= link_to "<i class='fa fa-edit'></i>".html_safe, '#', 'ng-click' => "suggestion_#{suggestion.id}_edit=true", onclick: 'return false;', class: 'list-link' %> 
           &nbsp; &bull; &nbsp;
-          <%= link_to "<i class='fa fa-trash'></i>".html_safe, suggestion_path(suggestion.id), method: :delete, data: { confirm: "Are you sure you want to delete your suggestion?"}, class: 'list-link' %>
+          <%= link_to "<i class='fa fa-trash'></i>".html_safe, suggestion_path(suggestion.id), method: :delete, data: { confirm: "#{t('suggestions.card._show.confirm')}"}, class: 'list-link' %>
         </small>
       <% end %>
     </h3>
   </div>
 
   <div class='col-sm-2'>
-    <p class='suggestion-time'><%= "#{time_ago_in_words(suggestion.created_at)} ago" %></p>
+    <p class='suggestion-time'>
+
+
+      
+
+<%= "#{t('suggestions.card._show.time', :time => time_ago_in_words(suggestion.created_at))}".titleize %>
```

Comment:
Minor code style nit:  lots of whitespace and the only thing that looks to be added was the `.titleize`.